src/TrueLayer/Payments/Model/ListPaymentRefundsResponse.cs

```diff
@@ -0,0 +1,8 @@
+using System.Collections.Generic;
+using OneOf;
+
+namespace TrueLayer.Payments.Model;
+
+using RefundUnion = OneOf<RefundPending, RefundAuthorized>;
+
+public record ListPaymentRefundsResponse(List<RefundUnion> Items);
```

Comment:
Along with the above comment, can we please rename this record type to `GetPaymentRefundsResponse`?

Reply:
Never mind, I can do it in my PR